frontend/src/pages/contacts/ContactStatusSelect.tsx

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import React, { useState } from 'react';
+import { SelectFormControl } from '../../components/molecules/formControls/selectFormControl';
+import { uri } from '../../utils';
+
+interface Props {
+	scoreDigit(results: any): void;
+}
+
+const ContactStatusSelect: React.FC<Props> = ({ scoreDigit }) => {
+	const [selectedStatus, setSelectedStatus] = useState<string>('All');
+	const [error, setError] = useState<string[] | null>(null);
+
+	const handleStatusChange = async (event: any) => {
+		const status = event.target.value;
+		setSelectedStatus(status);
+
+		let url = uri('contacts');
+		if (status !== 'All') {
+			url = uri(`contacts?status=${status}`);
+		}
+
+		try {
+			const response = await axios.get(url, {
+				headers: {
+					Authorization: `Bearer ${localStorage.getItem('access_token')}`,
+				},
+			});
+			scoreDigit(response.data);
+		} catch (err: any) {
+			setError(err?.response?.data?.message);
+			if (err.message) setError(err.message);
+		}
```

Comment:
You're setting the `setError` two times. It's better to put the first `setError` inside the `else` of the if.